DESIGN.md

```diff
@@ -191,7 +191,9 @@ classes live in `subiquity.models`.  An instance of each model class is
 attached as an attribute to the `SubiquityModel` class and the name of the
 attribute added to `INSTALL_MODEL_NAMES` or `POSTINSTALL_MODEL_NAMES` as
 appropriate. Models that go into `INSTALL_MODEL_NAMES` need to define a
-render() method that returns a fragment of curtin config.
+render() method that returns a fragment of curtin config.  POSTINSTALL models
+should define a make_cloudconfig() method that returns a cloud config
+fragment.
```

Comment:
I think this could be slightly more accurately phrased as "POSTINSTALL models that contribute to cloud-init config should define ..."

Thanks for remembering to update this file btw :)